tests/turbo.web.test.ts

```diff
@@ -374,32 +424,41 @@ describe('Browser environment', () => {
           })
           .catch((error) => error);
         expect(error).to.be.instanceOf(FailedRequestError);
-        expect(error?.message).to.equal('Failed request: 400: Bad Request');
+        expect(error?.message).to.equal(
+          "Failed request: 400: No promo code found with code 'BAD_PROMO_CODE'",
+        );
       });
     });
 
     describe('fund()', function () {
-      this.timeout(30_000); // Can take awhile for payment to retrieve transaction
-
-      // Skipped this test in CI because the provided fresh wallet is underfunded on arweave
-      // TODO: run arlocal in CI instead of using payment dev / arweave.net
-      // before(async() => await arweave.api.post('fund' ... ))
-      it.skip('should succeed', async () => {
-        const { winc } = await turbo.topUpWithTokens({
-          tokenAmount: WinstonToTokenAmount(10),
-        });
+      it('should succeed', async () => {
+        const delayedBlockMining = async () => {
+          let blocksMined = 0;
+          while (blocksMined < 3) {
+            await mineArLocalBlock();
+            blocksMined++;
+            await new Promise((resolve) => setTimeout(resolve, 10));
```

Comment:
nit: we use this in the src and tests - may be worth whipping up our beloved `sleep` function that wraps this

Reply:
40bb245